app/views/posts/index.html.erb

```diff
@@ -1 +1,6 @@
-<h1> Posts list </h1>
\ No newline at end of file
+<h1> Posts list </h1>
+<%= render partial: "shared/usercard", locals: { user: @user} %>
+<% @posts.each do |post|%>
+  <%= render partial: "shared/postcard", locals: { post: post} %>
+  <%= render partial: "shared/commentscard", locals: {comments: @comments.where(post_id: post.id).limit(5)}%>
```

Comment:
- [ ] Same here, Please use `post.recent_comments` to render 5 most recent comments because it's a project requirement.

![image](https://user-images.githubusercontent.com/68796232/180582027-6b0c99c6-2421-400c-a59a-cb7651b10e89.png)